### ISSUE TITLE
Game behaves differently depending on the runtime

### ISSUE THREAD

**Issue by BonzerKitten7** [Scirra/Construct-3-bugs#1810]
## Problem description

In my game the player have the ability to dash in any direction, and the ability works pretty well in the C2 runtime, but not in the C3 runtime, it's buggy for no reason.
I'll show you a sequence of dashs in various directions (→↑←) on both C2 and C3 runtime so you can see what i'm talking about.

C2 RT - Everything works well here, i press "x" + "→" and the player will dash to the right, i press "x" + "↑" and the player will dash upwards, i press "x" + "←" and the player will dash to the left.
https://i.imgur.com/Dx5jO9k.gif

C3 RT - Somethings are buggy here, a random black block appear on top of the player sometimes when he starts a dash (probably a webgl problem because i have a ball with some effects to distort the background when the player start the dash), the first dash is normal, but if you hit a wall the second dash will not work, and the third dash works, but the player go off the ground before starting the dash.
https://i.imgur.com/EhBAEH6.gif

All this weird behaviours start by simply changing from one runtime to another, definitely a bug.

I'll start to minimize my whole project so it's easier for you guys to debug it, i'll add the .c3p file here later today.

## Attach a .c3p

Working on it.

## Steps to reproduce

1. Open the game (C2 RT)
2. Press "x" + "→"
3. Press "x" + "↑"
4. Press "x" + "←"
5. Open the game (C3 RT)
6. Press "x" + "→"
7. Press "x" + "↑"
8. Press "x" + "←"

## Observed result

The player behaviour change according to the runtime.

## Expected result

The player behaviour to be always the same.

## Affected browsers

* Chrome: yes

## System details

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by AshleyScirra** [comment]
Please only file an issue once you have the .c3p. I'm afraid without the project there is nothing we can do - we can't debug gifs.

**Comment by BonzerKitten7** [comment]
I'm pretty sure you can't debug a .gif, that's why i said that i would attach the .c3p file later today. I started to write this issue today but i couldn't finish it because it really take some time for me since english is not my main language and GitHub doesn't allow me to save what i already wrote as a scratch, so i posted it without the .c3p file and said that i would attach it later, there's was literally no reason for you to close this issue since it's far from being solved. But thank you for your support anyway, i guess i'll need to open another issue when i have the time.
